### PR TITLE
add lexid

### PR DESCRIPTION
**Pull request by raybellwaves**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by raybellwaves** [comment]
not sure if windows failing is related to the PR: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=295073&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77

**Comment by raybellwaves** [comment]
@conda-forge-admin, please restart ci

**Comment by jakirkham** [comment]
As best as I can tell, there is a `/` that causes the Windows build to fail. Raised as issue ( https://github.com/mbarkhau/lexid/issues/1 )

**Comment by jakirkham** [comment]
Thanks for the upstream PR. Would suggest just including that here as a patch for now. Can drop it whenever the next release rolls around

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#patches

**Comment by mbarkhau** [comment]
Merged and cut a new release `lexid==2021.1006`. Cheers.

File: recipes/lexid/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% set name = "lexid" %}
-{% set version = "2020.1005" %}
+{% set version = "2020.1006" %}
```
Comment (mbarkhau) [inline recipes/lexid/meta.yaml:2]: I think you'll find, it's `2021.1006`

**Comment by jakirkham** [comment]
Awesome thanks @mbarkhau ! 😄

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Ray! 😀

**Pull request merged**